README.md

```diff
@@ -11,30 +12,41 @@ These are served via two configurable domain names, such as `docker.example.com`
 Requirements
 ============
 
-Python3 on the host is strongly recommended, as Ansible with Python 2 is deprecated and could break at any point.
-
-Hardware requirements for the instance are fairly low, requiring simply 1 core and 2GB RAM at a minimum. Disk requirements will vary depending on the number of images cached. 
-
-In testing mirroring all components for a Magnum cluster deployment required 30-70GB depending on mirrored version granularity.
-
-Naturally these requirements will scale with the number of users: in testing a single user doing a docker pull over LAN will consume 1 CPU for TCP handling. Thus, for 10 concurrent pulls 12+ cores will be required (to allow for various host overheads).
+- Kubernetes cluster 1.10+ (can use Cluster API)
+- Helm 2.8.0+
+- High available ingress controller
+- High available PostgreSQL database
+- High available Redis (Redis Sentinel) 
+- PVC that can be shared across nodes or external object storage
+- Install additional requirements 
+`ansible-galaxy install -r requirements.txt`
 
 Preparing to Deploy
 ===================
 
+- Create a virtual environment 
+`sudo apt install python3-venv`
+`python3 -m venv venv`
+`source venv/bin/activate`
 - In `roles/harbor_server/defaults/main` copy `secrets.yml.template` to `secrets.yml` and fill in as appropriate
 - In `playbooks/deploy_docker_mirror`, check the planned instance name and variables associated
 - Ensure that `defaults/main.yml` for the docker mirror role match the harbor role
 - If you want a volume for Harbor mount `/harbor_data` using `/etc/fstab` before deploying
 - Decide if SSL termination will be done upstream and set the var in the playbook as appropriate.
 - If you are using a docker hub token, create a file at `/opt/nginx/docker-mirror.env` with `REGISTRY_PROXY_USERNAME` set to the username and `REGISTRY_PROXY_PASSWORD` set to the access token.
+- If you are using postrgres in Kubernetes: 
+`export DB_PASSWORD=$(kubectl get secret --namespace default postgres-db-2-postgresql -o jsonpath="{.data.postgres-password}" | base64 -d) ` 
+`export DB_PASSWORD=<your DB password>`
+`export REDIS_PASSWORD=<your REDIS password>`
 
 With Host SSL
 -------------
 
 If you are planning on handling SSL termination on the server you must do the following too:
 - Issue a certificate to the harbor hostname and the docker mirror
 - Copy .pem files to `roles/docker_cache/files` as per the instructions inside. By default these are in .gitignore
+- Create TLS secret (chage order of the chain if certifications are from Digital Infrastructure)
+`kubectl create secret tls secret-tls --cert test.harbor.stfc.ac.uk.crt --key test.harbor.stfc.ac.uk.key -n harbor`
 
 Deploy / Reconfigure
 ====================
```

Comment:
We should add the step with the `ansible-playbook playbooks/<name>`

Reply:
I think this step is described in "Deploy / Reconfigure Section" - line 60